### PR TITLE
feat: support angular 9

### PR DESCRIPTION
**Pull request by janvennemann**
JIRA: https://jira.appcelerator.org/browse/FRAME-12

- [x] Update to Angular 9
- [x] Use new Webpack build from SDK 9.1.0
- [x] Update sample app

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,12 +1,16 @@
 export * from './directives';
 export * from './facades';
 export * from './log';
+export * from './mixins'
 export * from './platform/platform-titanium';
 export * from './platform/platform-titanium-dynamic';
 export * from './renderer';
-export * from './router';
+
+// export { TitaniumRouter, TitaniumRouterModule } from './router';
```
Comment (ewanharris) [inline src/index.ts:9]: Remove?

File: src/renderer/TitaniumRenderer.ts [hunk 2]
```diff
@@ -51,31 +48,28 @@ export class TitaniumRenderer extends Renderer2 {
...
+            const node = new TitaniumElement(name, elementEntry.resolveFactory(), elementEntry.meta)
             return node;
         }
 
         return new InvisibleElement(name);
     }
 
     createComment(value: string): CommentNode {
-        this.logger.debug(`TitaniumRenderer.createComment: ${value}`);
+        // this.logger.debug(`TitaniumRenderer.createComment: ${value}`);
```
Comment (ewanharris) [inline src/renderer/TitaniumRenderer.ts:63]: Is this kept in for debug purposes?

File: src/router/directives/TitaniumRouterLinkDirective.ts [hunk 3]
```diff
@@ -55,7 +55,8 @@ export class TitaniumRouterLinkDirective {
         };
         this.titaniumRouter.navigateByUrl(this.urlTree, options)
             .catch(e => {
-                this.logger.error(e.message);
+                console.log('RouterLinkDirectove onClick navigateByUrl');
```
Comment (ewanharris) [inline src/router/directives/TitaniumRouterLinkDirective.ts:58]: Is this log needed?

File: src/router/directives/TitaniumRouterOutletDirective.ts [hunk 4]
```diff
@@ -125,18 +129,18 @@ export class TitaniumRouterOutletDirective implements OnInit, OnDestroy {
         const factory = resolver.resolveComponentFactory(component);
         const childContexts = this.parentContexts.getOrCreateContext(this.name).children;
         const injector = new OutletInjector(activatedRoute, childContexts, this.location.injector);
+        const loaderRef = this.location.createComponent(this.detachedLoaderFactory, this.location.length, injector);
+        this.activated = loaderRef.instance.loadWithFactory(factory);
 
         if (this.isInitialRoute) {
-            this.activated = this.location.createComponent(factory, this.location.length, injector);
+            // this.activated = this.location.createComponent(factory, this.location.length, injector);
```
Comment (ewanharris) [inline src/router/directives/TitaniumRouterOutletDirective.ts:136]: Remove if not needed?

File: src/router/directives/TitaniumRouterOutletDirective.ts [hunk 5]
```diff
@@ -125,18 +129,18 @@ export class TitaniumRouterOutletDirective implements OnInit, OnDestroy {
...
             this.changeDetector.markForCheck();
-            this.triggerChangeDetection();
             this.navigationManager.createAndOpenRootNavigator(this.activated);
             this.isInitialRoute = false;
         } else {
-            const loaderRef = this.location.createComponent(this.detachedLoaderFactory, this.location.length, injector);
+            // const loaderRef = this.location.createComponent(this.detachedLoaderFactory, this.location.length, injector);
```
Comment (ewanharris) [inline src/router/directives/TitaniumRouterOutletDirective.ts:141]: Remove if not needed?

**Comment by janvennemann** [comment]
@ewanharris removed the debug logs and useless comments you found.

Any todo/fixme comments are for future work, yeah. I'm not sure if all are up-to-date so i need to go through them and create tickets for the ones that are still relevant.

**Pull request merged**